config/initializers/idv_proofer.rb

```diff
@@ -1,2 +1 @@
-Dir[Rails.root.join('lib', 'proofer_mocks', '*')].sort.each { |file| require file }
```

Comment:
did this get replaced by something? Or is it that the directory doesn't exist anymore?

Reply:
Does not exist 🙂